### ISSUE TITLE
Add latency meter to the [tab] player display (and on server list page)

### ISSUE THREAD

**Issue by Cervator** [MovingBlocks/Terasology#2377]
At present the player list you get when holding down `tab` only lists names. It would be good to list a little more info, like the player's latency to the server. Anything else we could think about adding? Maybe also a player summary (count) at the very top.

With a latency meter available we could add it to the server listing UI as well.

Linking to #1099 which also talks about the server listing UI (is missing player count)
##

**Comment by jpollmann102** [comment]
Hi, I'm pretty new to helping out open source projects and I'm not that great with Github, but I'd like to get to know it and what not. This seems like something I could possibly give a crack at, any tips on where to start, module wise?

**Comment by Cervator** [comment]
Related: #2602 was merged so there is a ping command in-game now. Maybe that could be put to use for the latency meter too?

**Comment by iojw** [comment]
I took a look at this and managed to implement a rough version based on the code from the ping command

![capture](https://cloud.githubusercontent.com/assets/1729493/20965046/1412e798-bcaf-11e6-848c-50250db15b49.PNG)

Opening the socket on the main UI thread appears to make the game slow down and freeze a little bit, so I opened up a new thread for retrieving the ping. Is that alright? If so, it shouldn't be hard to apply the same thing to the player overlay as well. 🙂
